src/app/api/healthcheck/route.ts

```diff
@@ -0,0 +1,46 @@
+import { NextResponse } from "next/server";
+import { db } from "@/lib/db";
+
+/**
+ * Health check endpoint for container orchestration and load balancers.
+ * Checks database connectivity and returns service status.
+ */
+export async function GET() {
+  const startTime = Date.now();
+
+  try {
+    // Check database connectivity
+    await db.$queryRaw`SELECT 1`;
+
+    const responseTime = Date.now() - startTime;
+
+    return NextResponse.json(
+      {
+        status: "healthy",
+        timestamp: new Date().toISOString(),
+        uptime: process.uptime(),
+        responseTime: `${responseTime}ms`,
+        database: "connected",
+        version: process.env.npm_package_version || "unknown",
```

Comment:
This is not required. All other are cool!

Reply:
Got it 👍 I’ve removed npm_package_version. Everything else stays as is.